data/achievements/season_01.yaml

```diff
@@ -0,0 +1,114 @@
+- title: Three Scoops
+  side: corp
+  points: 1
+  description: Install three pieces of ICE on the same server in one turn.
+---
```

Comment:
The structure of a single achievement should be clear. You could break the file into distinct `corp` and `runner` files if you like.